README.md

```diff
@@ -41,10 +41,12 @@ The `samlconf` script may take the following parameters:
                 the default implementation directory is /implementations/samlconf-ddf-impl.
                       
            -d | --debug
-               Sets the log level to debug.
+                Sets the log level to debug.
 
-           -e | --error
-               Run tests that expect errors.
+           -l | --lenient
+                When an error occurs, the SAML V2.0 Standard Specification requires an IdP to respond with a valid SAML response containing an error <StatusCode>.
+                The SAML Specification does not allow responding with error HTTP status codes however, if the -l flag is given, this test kit will allow HTTP error status codes as a valid error response.
+                If it is not given, this test kit will only verify that a valid SAML error response is returned.
```

Comment:
✏️ Could move this sentence to be right after the first sentence and change this sentence to `That is what the test kit will look for, by default.` Then the second sentence would talk about how adding the `-l` option also allows for HTTP error codes (i.e. 400's and 500's).